### DISCUSSION TITLE
Retrieve from vectorDB (Milvus) as a tool in Bee agent framework

### DISCUSSION THREAD

**Discussion by SatishVenkatanarayana** [i-am-bee/beeai-framework#157] [General]
Like we have SQL tool, can we have a tool to retrieve data from Milvus based on a query?

**Comment by Tomas2D** [comment]
That would be great. Issue created https://github.com/i-am-bee/bee-agent-framework/issues/158

**Comment by TechcodexIO** [comment]
If we are going to add Milvus and Qdrant, we might as well consider LanceDB as well.  I have worked with all three extensively and can look further in to this. Should be some level of abstraction since most use cases for vectorDB in the context of agents and GenAI are the same across the board.

**Reply by Tomas2D** [reply]
It would be useful to have a unified experience or at least some base interface.

**Comment by Tomas2D** [comment]
Released in v0.0.44

**Comment by SatishVenkatanarayana** [comment]
Thanks a lot for the quick resolution of this!!

________________________________
From: Tomáš Dvořák ***@***.***>
Sent: Friday, November 29, 2024 10:03:56 PM
To: i-am-bee/bee-agent-framework ***@***.***>
Cc: Satish Venkatanarayana ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [Bee] Retrieve from vectorDB (Milvus) as a tool in Bee agent framework (Discussion #157)

Closed #157 as resolved. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread. Message ID: <i-am-bee/bee-agent-framework/repo-discussions/157/discussion_event/1646485@ github. com>


Closed #157<https://github.com/orgs/i-am-bee/discussions/157 > as resolved.
